DataStructure/LinkList/linklist.java

```diff
@@ -0,0 +1,455 @@
+package LinkList;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class linklist {
```

Comment:
It should be linkedList instead of linklist. Correct this throughout the file. And in file and folder naming.